Dockerfile

```
@@ -0,0 +1,22 @@
+FROM ubuntu:24.04
+
+# wget is here for container healthchecks (GET /_health); nothing in the server
+# needs it.
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)